RELEASE/scripts/autoscend/combat/auto_combat_quest.ash

```diff
@@ -82,23 +82,46 @@ string auto_JunkyardCombatHandler(int round, monster enemy, string text)
 		return useSkill($skill[Good Medicine]);
 	}
 
-	if (my_location() != $location[The Battlefield (Frat Uniform)] && my_location() != $location[The Battlefield (Hippy Uniform)] && !get_property("auto_ignoreFlyer").to_boolean())
+	item flyer = $item[Rock Band Flyers];
+	if(auto_warSide() == "hippy")
 	{
-		if (canUse($item[Rock Band Flyers]) && get_property("flyeredML").to_int() < 10000)
+		flyer = $item[Jam Band Flyers];
+	}
+	skill stunner = getStunner(enemy);
+	boolean stunned = combat_status_check("stunned");
+	
+	if (get_property("auto_gremlinMoly").to_boolean() && !canSurvive(20) && !stunned)		//don't flyer tool gremlins if it's dangerous to survive them for long
+	{
+		if(monster_attack() > ( my_buffedstat($stat[moxie]) + 10) && !canSurvive(10) && haveUsed($skill[Curse Of Weaksauce]))
 		{
-			if (isActuallyEd())
+			//if after all deleveling it's still too strong to safely stasis let weaksauce delevel it more in exchange for a few turns
+			//except if stuck with an attack familiar or unforeseen passive damage effects that can kill the gremlin
+			boolean gremlinTakesDamage = (isAttackFamiliar(my_familiar()) || (monster_hp() < (0.8*monster_hp(enemy))));
+			if(!gremlinTakesDamage && round < 10 && stunner != $skill[none])
 			{
-				set_property("auto_edStatus", "UNDYING!");
+				combat_status_add(",stunned");
```

Comment:
```suggestion
				combat_status_add("stunned");
```
combat_status_add automatically handles the syntax. also I undid the the switch to comma deliminated and made it stay in the current paradigm of wrapping each skill used with ()